# HG changeset patch
# User David Bateman <dbateman@free.fr>
# Date 1227166932 -3600
#      Thu Nov 20 08:42:12 2008 +0100
# Branch release-3-0-x
# Node ID 8dc2fa08600c4099d56dafa229b8d80610bd154e
# Parent  84fb5019fb383147a97c5aaf87b2d72500b0f9e2
Changelog for previous patch

diff --git a/liboctave/ChangeLog b/liboctave/ChangeLog
--- a/liboctave/ChangeLog
+++ b/liboctave/ChangeLog
@@ -1,8 +1,14 @@
+2008-11-18  David Bateman  <dbateman@free.fr>
+
+	* file-ops.cc (std::string file_ops::tilde_expand (const 
+	std::string&)): Check if the string contains a tilde and fast 
+	return if not.
+
 2008-11-17  John W. Eaton  <jwe@octave.org>
 
 	* dir-ops.cc (dir_entry::read): Use std::list<std::string> to
 	cache names before converting to string_vector.
 
 2008-10-09  Jaroslav Hajek <highegg@gmail.com>
 
 	* oct-sort.cc (octave_sort<T>::merge_getmem,
